experimental/segmenter/src/complex.rs

```diff
@@ -121,42 +121,23 @@ impl Dictionary {
     >(
         provider: &D,
     ) -> Self {
-        let cj = Self::load_chinese_japanese(provider, locale!("ja")).ok();
+        let cj = Self::load_w_auto(provider, locale!("ja")).ok();
         Dictionary {
             cj,
             ..Default::default()
         }
     }
 
-    pub(crate) fn load_chinese_japanese<
-        D: DataProvider<DictionaryForWordOnlyAutoV1Marker> + ?Sized,
-    >(
-        provider: &D,
-        locale: Locale,
-    ) -> Result<DataPayload<UCharDictionaryBreakDataV1Marker>, DataError> {
-        provider
-            .load(DataRequest {
-                locale: &DataLocale::from(locale),
-                metadata: {
-                    let mut m = DataRequestMetadata::default();
-                    m.silent = true;
```

Comment:
Question: we have `m.silent = true` here, but the new `load_w_auto` doesn't set it. Is it intentional?

Reply:
I think yes! Merge problems. I'll fix.